jax2onnx/converter/ir_postprocess.py

```diff
@@ -0,0 +1,275 @@
+# jax2onnx/converter/ir_postprocess.py
+
+from __future__ import annotations
+
+from collections.abc import Iterable as IterableABC
+from typing import Iterable, Sequence, Any
+
+import numpy as np
+import onnx_ir as ir
+from onnx_ir import AttributeType as IRAttrType
+
+
+def _list_from_maybe_iterable(obj: object) -> list[Any]:
+    if obj is None:
+        return []
+    if isinstance(obj, (str, bytes)):
+        return []
+    if isinstance(obj, IterableABC):
+        try:
+            return list(obj)
+        except Exception:
+            return []
+    return []
+
+
+def _value_name(value: object) -> str | None:
+    if isinstance(value, str):
+        return value or None
+    return getattr(value, "name", None)
```

Comment:
It would probably be a good idea to remove all `getattr` calls from this file and instead us the correct API defined on the IR classes